scripts/serve-local.sh

```diff
@@ -15,7 +15,10 @@ docker run \
     --volume "$PWD/src:/app/src" \
     --volume "$PWD/public:/app/public" \
     --volume "$PWD/package.json:/app/package.json" \
+    --volume "$PWD/static.config.js:/app/static.config.js" \
+    --volume "$PWD/.babelrc:/app/.babelrc" \
     --volume "$PWD/yarn.lock:/app/yarn.lock" \
     --env "REACT_APP_CMS_ENDPOINT=http://localhost:8000/api" \
+    --env "API_URL=$(ifconfig en0 | awk '$1 == "inet" {print $2}')" \
     --env "REACT_APP_CMS_ASSETS=http://localhost:8000/media" \
```

Comment:
Do we need this URL anymore? I don't think `localhost` will work inside the container but might be fine for loading images.